frontend/src/components/kl-add-item.js

```diff
@@ -28,7 +28,18 @@ export default {
             }).catch(err => {
                 if (err.status === 400 && err.body.code === 1) {
                     let renderedErrors = renderValidationErrors(err.body.errors);
-                    renderedErrors.forEach(error => toastr.error(this.$t('errors.' + error.id, error.props)));
+                    renderedErrors.forEach(error => {
+                        if (error.id === 'mustBeLessThan')
+                            toastr.error(this.$t('errors.' + error.id, error.props));
+                        else {
+                            let res;
+                            if (error.count === 1)
+                                res = 1;
```

Comment:
And using `t` both here and in `renderValidationErrors` is not good. I remember telling you that a while ago, and I'll repeat - you should only be using `this.$t`.